src/server.js

```diff
@@ -76,6 +86,8 @@ const main = async () => {
   app.post('/api/event', promiseHandler(withAuth(async req => {
     const { event, properties } = req.body
 
+    validateInput(req.body, Joi.object().keys({ event: eventSchema, properties: propertiesSchema }))
```

Comment:
It looks like Terra UI is using the key `data` instead of `properties`. Not sure which should change. Also, Terra UI wasn't sending a `userId` in my testing. No fault in this PR, but it's something we should reconcile soon.

Reply:
Ultimately mixpanel is expecting properties. My vote would be for the UI to send along properties to minimize data transformations.